Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs

```diff
@@ -27,6 +27,10 @@ protected override void OnModelCreating(ModelBuilder builder)
         public DbSet<Category> Category { get; set; }
         public DbSet<SingleMultipleAnswerQuestion> SingleMultipleAnswerQuestion { get; set; }
         public DbSet<SingleMultipleAnswerQuestionOption> SingleMultipleAnswerQuestionOption { get; set; }
+		
+		public DbSet<CodeSnippetQuestion> CodeSnippetQuestion { get; set; }
```

Comment:
Spacing seems to be proper on local branch. I will commit directly on the remote branch.